src/main/java/org/gridsuite/loadflow/server/LoadFlowController.java

```diff
@@ -157,9 +157,12 @@ public ResponseEntity<Map<String, List<com.powsybl.commons.parameters.Parameter>
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The limit violations")})
     public ResponseEntity<List<LimitViolationInfos>> getLimitViolations(@Parameter(description = "Result UUID") @PathVariable("resultUuid") UUID resultUuid,
                                                                         @Parameter(description = "Filters") @RequestParam(name = "filters", required = false) String stringFilters,
-                                                                        @Parameter(description = "Sort parameters") Sort sort) {
+                                                                        @Parameter(description = "Global Filters") @RequestParam(name = "globalFilters", required = false) String globalFilters,
+                                                                        @Parameter(description = "Sort parameters") Sort sort,
+                                                                        @Parameter(description = "network Uuid") @RequestParam(name = "networkUuid", required = false) UUID networkUuid) {
```

Comment:
Need to pass the varaiantId

Reply:
we pass the networkUUID because we need to evaluate filter but why we should add the variantId?